### DISCUSSION TITLE
Confusion regarding re-verification of student status on Github

### DISCUSSION THREAD

**Discussion by Linzh7** [community/community#54142] [GitHub Education] [answered]
Hi,

I'm a student and I've been using Github for my projects and researches. 

Recently, I've been receiving emails stating that I've lost my student status, and Github has also sent me an email asking me to reverify my status. However, when I tried to reverify my status on the website, I received a message saying that I do not need to reverify at this time.

`Hi, Linzh! You were last verified as a student on Mar 15, 2022. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits.`

I'm confused about what to do now. May I ask for some help to guide me on how to reverify my student status on Github? I would appreciate any help or advice on this matter.

Thank you so much for your time and support.

Best regards,
Linzh

**Comment by hrishikesh-kocharekar-dev** [comment, marked as answer]
Hi Linzh,

It sounds like there might be some confusion regarding your student status verification on Github. It's possible that the emails you received are part of a routine check to ensure that only eligible students are using Github's student benefits.

Since you received a message saying that you do not need to reverify your status at this time, it's likely that your verification is still valid and there is no need for you to take any further action.

However, if you are still unsure about your verification status, I recommend that you reach out to Github's support team directly. They will be able to provide you with accurate information and help you resolve any issues with your verification status.

To contact Github's support team, you can visit their Help page and submit a request through the contact form. You can also try tweeting to their official Twitter account or contacting them through their Facebook page.

**Comment by mohitvdx** [comment]
Hi @hrishikesh-kocharekar-dev I received a similar mail asking me to reverify my github student status. I did all the steps but my application got rejected because I don't have a Institution Email address.

Reason is that I am a freshman and apparently my institute provides an email address a bit late.

How should I proceed?

**Reply by Mel28vin** [reply]
Applying with my fee receipt worked @mohitvdx

**Comment by pratyusshh** [comment]
> as also sent me an email asking me to reverify my status. However, when I tried to reverify my status on the website, I received a message saying that I do not need to reverify at this time.
> 
> `Hi, Linzh! You were last verified as a student on Mar 15, 2022. It is not`

i got the mail , how should i proceed please help??
